test/verify/check-bots-api

```diff
@@ -94,6 +94,19 @@ class TestImageCustomize(unittest.TestCase):
 
         self.checkBoot(img)
 
+    def testFailurePropagation(self):
+        img = os.environ["TEST_OS"]
+        with testvm.Timeout(seconds=300, error_message="Timed out waiting for image-customize"):
+            subprocess.check_call(["bots/image-customize", "--verbose",
+                                   "--run-command", "true", img])
+
+            try:
+                subprocess.check_call(["bots/image-customize", "--verbose",
+                                       "--run-command", "false", img])
+                raise Exception("subprocess.check_call should have failed, but didn't")
+            except subprocess.CalledProcessError:
```

Comment:
Hint: This can be written more elegantly with [assertRaises()](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises).